### ISSUE TITLE
Print result does not work

### ISSUE THREAD

**Issue by Dellos7** [deckgo/deckdeckgo#47]
Hello @peterpeterparker,

I was just trying the print result button but it seems not to be working, at least from my computer, as shown in this screenshot:

![deckdeckgo print](https://i.imgur.com/mZOz6z9.jpg)

Result is just a blank page. Could you please confirm this?

I've tried it in Chrome and Safari.

Thank you.

David.

**Comment by peterpeterparker** [comment]
@Dellos7 well spotted, probably one of the last CSS changes is breaking it

**Comment by peterpeterparker** [comment]
Yep definitely a style of `ion-content` is blocking the print feature
